node_lambnik/src/proxy/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM nginx:1.12
+
+COPY etc/nginx/conf.d/default.conf /etc/nginx/conf.d/default.conf
```

Comment:
Rather than reproduce the whole directory structure here, you could just put the conf file in this directory with the `Dockerfile` and change this line to `COPY nginx.conf /etc/nginx/conf.d/default.conf`.  Works the same and seems cleaner to me.